### PR TITLE
Improved Account Settings Documentation

### PR DESCRIPTION
**Pull request by DavidLu1997**
- Added the fact that profile pictures are cropped circularly
- Added that support for more languages is being added

### CODE REVIEW

File: source/help/settings/account-settings.md [hunk 1]
```diff
@@ -88,14 +88,16 @@ Configure how names are displayed in Mattermost: nickname, username or full name
...
-Select what language Mattermost displays in the user interface. Options include: 
+Select what language Mattermost displays in the user interface. Options currently include: 
 
 - English
 - Spanish
 - Portuguese
 - French
 - Japanese
 
+More languages are being added, please check back later if your preferred language is not on the list.
```
Comment (esethna) [inline source/help/settings/account-settings.md]: Propose instead: 

"If you’re interested in contributing translations to Mattermost, please read our documentation on [localization](http://docs.mattermost.com/developer/localization.html)."

**Comment by esethna** [comment]
Otherwise looks good! Thanks @DavidLu1997

File: source/help/settings/account-settings.md [hunk 2]
```diff
@@ -88,14 +88,16 @@ Configure how names are displayed in Mattermost: nickname, username or full name
 Select if the text in the center channel is fixed width and centered, or full width. 
 
 #### Language
-Select what language Mattermost displays in the user interface. Options include: 
+Select what language Mattermost displays in the user interface. Options currently include:
```
Comment (it33) [inline source/help/settings/account-settings.md]: Don't think "currently" or "check back later" is useful for end user documentation, this is installed software, not a SaaS service.

**Comment by it33** [comment]
Not helpful. This is installed software, not a SaaS service. The languages in the product don't change until the next upgrade.

**Pull request closed without merging**